### ISSUE TITLE
Rename has problem

### ISSUE THREAD

**Issue by TDCBQ** [willscott/go-nfs#69]
No metadata update after rename

![image](https://github.com/willscott/go-nfs/assets/37407560/e2307d02-91da-42a4-bb42-459d894dbfde)

**Comment by willscott** [comment]
can you provide details on the client OS where you're seeing this and how the mount is mounted?

**Comment by willscott** [comment]
I wonder if this is fixed through the invalidation in https://github.com/willscott/go-nfs/compare/master...evgeniiz321:go-nfs:fix-caching-handler 
@evgeniiz321 - do you feel like your extension of the caching handler was effective?

**Comment by DUOLabs333** [comment]
Could you check whether https://github.com/DUOLabs333/go-nfs/tree/patch-2 fixes your problems --- there's a couple of commits aimed at fixed onRename,.

**Comment by rminnich** [comment]
I did just try this, using github.com/u-root/sidecore, with nfs3 server, and it seems ok now:
cp /etc/hosts .
mv hosts hosts2
cat and ls -l show expected values.
```
~/go/src/github.com/u-root/sidecore/cmds/sidecore # cp /etc/hosts .
~/go/src/github.com/u-root/sidecore/cmds/sidecore # ls -l hosts
-rw-r--r--    1 502      dialout        174 Dec 31  2023 hosts
~/go/src/github.com/u-root/sidecore/cmds/sidecore # head -1 hosts
127.0.0.1	localhost
~/go/src/github.com/u-root/sidecore/cmds/sidecore # mv hosts hosts4
~/go/src/github.com/u-root/sidecore/cmds/sidecore # head -1 hosts4
127.0.0.1	localhost
~/go/src/github.com/u-root/sidecore/cmds/sidecore # ls -l hosts4
-rw-r--r--    1 502      dialout        174 Dec 31  2023 hosts4

127.0.0.1:e4c034e0-bc45-4763-bf09-8b15edcb5665/Users on /Users type nfs (rw,relatime,vers=3,rsize=1048576,wsize=1048576,namlen=255,hard,nolock,proto=tcp,port=45613,timeo=600,retrans=2,sec=sys,mountaddr=127.0.0.1,mountvers=3,mountport=45613,mountproto=tcp,local_lock=all,addr=127.0.0.1)
```

**Comment by willscott** [comment]
fixed in #118
